src/components/MapComponents/Map.js

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import mapboxgl from 'mapbox-gl';
+import {connect} from 'react-redux';
+import {UpdateCordinates} from "../../redux/actions/mapActions";
+import {MapPresentational} from './MapPresentational';
+import GetLocationData from '../LocationComponents/LocationCordinates.json';
+mapboxgl.accessToken = 'pk.eyJ1IjoibWFwYm94IiwiYSI6ImNpejY4M29iazA2Z2gycXA4N2pmbDZmangifQ.-g_vE53SD2WrJ6tFX7QHmA';
+
+
+let LatNLong='19,13';
+let locationList;
+class Map extends React.Component {
+ map;
+ mapRef=React.createRef();
```

Comment:
Creating refs is allowed. but its an anti-pattern because it adds one more dependency to keep track of and its lifecycle should be managed manually.